about.html

```diff
@@ -11,58 +11,77 @@
 
 </head>
 
-<body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
+<body class="about-page">
+    <div class="container-about-page">
+      <header>
+        <nav class="about-nav">
```

Comment:
You can also just target nav a { } instead of assigning each anchor a class name.